CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Features
 
+- [plutonium#1240](https://github.com/influxdata/plutonium/issues/1240): Add 'X-Influxdb-Build' to http response headers so users can identify if a response is from an OSS or Enterprise service.  
```

Comment:
This should be an `influxdata/influxdb` issue if there is one available, or if there isn't then it's usual for us to just link to this PR, e.g., 

```
[#8574](https://github.com/influxdata/influxdb/pull/8574): Add 'X-Influxdb-Build' to HTTP response headers so users can identify if a response is from an OSS or Enterprise service.  
```